tests/integration/targets/omevv_vcenter_info/tests/invalid-12250.yaml

```diff
@@ -88,9 +87,8 @@
       ignore_errors: true
 
     - ansible.builtin.set_fact:
-        invalid_validate_certs_expected_msg: "argument 'validate_certs' is of
-          type <class 'str'> and we were unable to convert to bool: The value
-          'invalid' is not a valid boolean.  Valid booleans include: "
+        invalid_validate_certs_expected_msg: "we were unable to convert to bool:
```

Comment:
Is this working for 2.18 as well?

Reply:
Yes, It's working with both versions. have added the results screenshot.